### ISSUE TITLE
SUPIR to use FLUX instead SDXL models ?

### ISSUE THREAD

**Issue by web2299** [kijai/ComfyUI-SUPIR#159]
I wonder if it will be possible SUPIR to use Flux instead SDXL models ? If this is already possible can someone share workflow please ? Thanks in advance !

**Comment by kijai** [comment]
No, SUPIR is basically a custom controlnet for SDXL, it won't work with anything else.

**Comment by web2299** [comment]
**kijai**  thanks for the answer !

**Comment by Metatronovich** [comment]
> I wonder if it will be possible SUPIR to use Flux instead SDXL models ? If this is already possible can someone share workflow please ? Thanks in advance !

This seems like overly complicated but it claims success on the subject: https://www.youtube.com/watch?v=ExF471Q1Goo
hope to get a comment on the subject from @kijai as well

**Comment by ZeroCool22** [comment]
> > > I wonder if it will be possible SUPIR to use Flux instead SDXL models ? If this is already possible can someone share workflow please ? Thanks in advance !
> > 
> > 
> > This seems like overly complicated but it claims success on the subject: https://www.youtube.com/watch?v=ExF471Q1Goo hope to get a comment on the subject from @kijai as well
> 
> Of course you can still use SUPIR normally with Flux outputs, you just can't use the Flux model in the SUPIR pipeline itself as it's for SDXL.
> 
> There's also this node: https://github.com/city96/SD-Latent-Interposer
> 
> Which is potentially helpful as it allows using Flux latents more directly into SUPIR, removing the need to decode/encode between the pipelines.

Can you provide a workflow ready to use?
